packages/ansible-language-server/src/providers/completionProviderUtils.ts

```diff
@@ -45,7 +45,7 @@ export function getVarsCompletion(
       ) {
         path = parentKeyPath;
         const scopedNode = path[path.length - 3].toJSON();
-        if (scopedNode.hasOwnProperty("vars")) {
+        if (Object.prototype.hasOwnProperty.call(scopedNode, "vars")) {
```

Comment:
@ssbarnea there is already a function in `utils/misc.ts` called `hasOwnProperty`. We can use that here at least for language-server related checks

Reply:
Also is there a way we can share this with extension code?

Reply:
It seems that using that function confuses ts compiler and gives errors. If you can manage to make it work in a follow-up it would be happy to use it as it would look less verbose.